dnf-docker-test/features/module-install-pkg-2.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Installing package from module - error handling
+
+  @setup
+  Scenario: Testing repository Setup
+      Given I run steps from file "modularity-repo-3.setup"
+       When I enable repository "modularityY"
+        And I successfully run "dnf makecache"
+
+  Scenario: I cannot install a specific package from a disabled module
```

Comment:
Should be called: I cannot install a specific package from a disabled module when default stream is not defined.

Reply:
Please also cover the scenario:
Having default stream defined I can install a specific package from a disabled module (see my email for details).